### ISSUE TITLE
Remove Support Chat from sidebar unless user has active chat session

### ISSUE THREAD

**Issue by kriskarkoski** [tinkertinker/wp-calypso#54]
In preparation for the beta of Happychat we need to tweak how users access chat to match how Olark is currently implements so we can staff both platforms while Happychat is in testing.

To do this, Happychat chat sessions should be started only from the wordpress.com/help/contact page (which is linked from wordpress.com/help, which is accessed from the Help option at the bottom of the Calypso sidebar). The Support Chat option currently shown in Calypso on all pages should be hidden and only shown if the user has an active chat session so that they can reopen chat with us if they've closed it for some reason.

It's not essential, but it would be nice to have the Support Chat link show in a different color than black (maybe orange) if they have a new message from us that they haven't seen yet.

This change should also make initial implementation of https://github.com/Automattic/happychat/issues/16 more straightforward since it can use a full page form like we do now on /help/contact.

![support chat highlighted](https://cloud.githubusercontent.com/assets/1900444/18894029/7df314f2-84de-11e6-9c90-acde589b74ab.jpg)

**Comment by andreabadgley** [comment]
Dependent on https://github.com/Automattic/happychat/issues/108.

**Comment by jasmussen** [comment]
Also dependent on #34, as that might have us move to a collapsed panel.

**Comment by jasmussen** [comment]
Dependant tickets have been merged, so closing this!
